testsuite/tests/kuadrant/gateway/conftest.py

```diff
@@ -0,0 +1,17 @@
+"""Conftest for gateway tests"""
+import pytest
+
+from testsuite.openshift.objects.auth_config.auth_policy import AuthPolicy
+
+
```

Comment:
There is a missing skip if Kuadrant is not enabled, see operator tests

Reply:
I believe we only have skip if Kuadrant IS enabled and this fixture enables Kuadrant tests by default. Authorino operator tests get disabled if Kuadrant is enabled. Can you please send a link to what you referring to?

Reply:
Sorry, you are right, The correct example is in limitador tests